packages/transactions/src/configs.ts

```diff
@@ -35,13 +35,64 @@ export const FAILED_REQUIRE_SIGNAL = '0xffffffffffff0000';
 // Revert with this value for a failing call to `std::asset::transfer_to_address`.
 export const FAILED_TRANSFER_TO_ADDRESS_SIGNAL = '0xffffffffffff0001';
 
-// Revert with this value for a failing call to `std::message::send_message`.
-export const FAILED_SEND_MESSAGE_SIGNAL = '0xffffffffffff0002';
```

Comment:
Why was this removed?

Reply:
@Dhaiwat10 This revert type was removed from Sway